### PR TITLE
Feature: Add backend support for Solana,Tezos in _load_account

### PR DESCRIPTION
**Pull request by hoh**
This refactors the structure of the helper `_load_account` used in the CLI and adds backend support for Solana and Thezos accounts.

### CODE REVIEW

**Comment by hoh** [comment]
@odesenfans is this going too far into typing ? I am not sure how to fix these `mypy` issues:

```python
src/aleph_client/account.py:24: error: Incompatible return value type (got "Union[ETHAccount, SOLAccount, TezosAccount]", expected "T")
src/aleph_client/account.py:30: error: Incompatible return value type (got "Union[ETHAccount, SOLAccount, TezosAccount]", expected "T")
src/aleph_client/account.py:49: error: Value of type variable "T" of "account_from_hex_string" cannot be "BaseAccount"
src/aleph_client/account.py:49: error: Incompatible return value type (got "BaseAccount", expected "Union[ETHAccount, SOLAccount, TezosAccount]")
src/aleph_client/account.py:52: error: Value of type variable "T" of "account_from_file" cannot be "BaseAccount"
src/aleph_client/account.py:52: error: Incompatible return value type (got "BaseAccount", expected "Union[ETHAccount, SOLAccount, TezosAccount]")
```

**Comment by MHHukiewitz** [comment]
After you made BaseAccount an ABC, lines 49 and 52 should be fixed, and maybe the rest too. Try merging in master and rerunning the checks now.

**Comment by hoh** [comment]
> After you made BaseAccount an ABC, lines 49 and 52 should be fixed, and maybe the rest too. Try merging in master and rerunning the checks now.

Rebased on master.

File: src/aleph_client/account.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+                unix_socket=settings.REMOTE_CRYPTO_UNIX_SOCKET,
+            )
+        )
+    else:
+        new_private_key = get_fallback_private_key()
+        account = account_type(private_key=new_private_key)
+        logger.info(
+            f"Generated fallback private key with address {account.get_address()}"
+        )
+        return account
```
Comment (odesenfans) [inline src/aleph_client/account.py:64]: We should have tests to load typical accounts.

**Review by odesenfans — APPROVED** [review]
I would add tests, but that can be added in a second PR.

**Pull request merged**